src/table_functions.cpp

```diff
@@ -22,7 +25,11 @@ auto static try_value_is_int(Value s, const std::string& method_name, int arg_in
     auto tmp = Number(1);
     try {
         if (s.is_string()) {
-            tmp = std::get<Number>(s.to_number());
+            Value v = s.to_number();
+            if (v == Nil()) {
+                throw std::runtime_error("");
```

Comment:
You should give an error message here. Also I think the implementation of `std::get<Number>` already gives an error message that the type is not a number. Maybe that is enought?

Reply:
The throw is only there to trigger an error for the try catch. and there is an error-message. so why should I add an error  message that gets ignored anyway?

Reply:
ok then it's fine